ggcore/utils.py

```diff
@@ -1,5 +1,9 @@
+"""Define util constants and enums."""
+
 import enum
 
+# pylint: disable=pointless-string-statement
+
 """GG MODULE CLIENTS"""
```

Comment:
Couldn't these be code comments rather than docstrings?
docstrings themselves are treated as strings themselves so this is equivalent to having a string `"GG MODULE CLIENTS"` within a file.